### PR TITLE
RATIS-672. Vagrant test's Maven 404's 

### PR DESCRIPTION
**Pull request by cbaenziger**
This fixes the Maven URL for the Vagrant tests.

### CODE REVIEW

**Comment by cbaenziger** [comment]
The most recent push validates all checksums and pins versions.

**Comment by cbaenziger** [comment]
This push ensures {{docker}} image and {{vagrant}} build both checksum downloaded artifacts and share URLs to provide one place to ensure one does not grow stale.

**Comment by joshelser** [comment]
Building the vagrant vm, I see:
```
==> ratis-build: Running provisioner: shell...
    ratis-build: Running: script: Build Namazu
    ratis-build: # runtime
    ratis-build: /usr/local/go/src/runtime/stubs_x86.go:10:6: stackcheck redeclared in this block
    ratis-build: 	previous declaration at /usr/local/go/src/runtime/stubs_amd64x.go:10:6
    ratis-build: /usr/local/go/src/runtime/unaligned1.go:11:6: readUnaligned32 redeclared in this block
    ratis-build: 	previous declaration at /usr/local/go/src/runtime/alg.go:321:40
    ratis-build: /usr/local/go/src/runtime/unaligned1.go:15:6: readUnaligned64 redeclared in this block
    ratis-build: 	previous declaration at /usr/local/go/src/runtime/alg.go:329:40
```

Maybe Go 1.13 doesn't work with this version of Namazu? The Namazu readme says they test against Go 1.6 (outdated docs? That appears to be a super old Go verison). Looks like we were on 1.11.2 previously.

**Comment by cbaenziger** [comment]
Huh! I see Namazu building with warnings indeed but not failing for me. I have dialed up the I/O request failure probability and see Namazu is indeed injecting successfully and I see I/O response times are being varied as requested.
```
vagrant@ratis-build:~$       cd namazu
vagrant@ratis-build:~/namazu$       export GOROOT=/usr/local/go
vagrant@ratis-build:~/namazu$       export GOPATH=`pwd`
vagrant@ratis-build:~/namazu$       # for some reason seelog fails to pull automatically
vagrant@ratis-build:~/namazu$       # Go should check its checksum database on go get per
vagrant@ratis-build:~/namazu$       # https://blog.golang.org/module-mirror-launch
vagrant@ratis-build:~/namazu$       go get -u github.com/cihub/seelog
vagrant@ratis-build:~/namazu$       ./build
[INFO] Installing protoc to /home/vagrant/namazu/bin
[INFO] Generating PB files (FIXME: skipped...)
[INFO] Building nmz
# github.com/osrg/namazu/nmz
/tmp/go-link-045955592/000009.o: In function `mygetgrouplist':
/workdir/go/src/os/user/getgrouplist_unix.go:16: warning: Using 'getgrouplist' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrgid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:38: warning: Using 'getgrgid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:43: warning: Using 'getgrnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:33: warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwuid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:28: warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000004.o: In function `_cgo_26061493d47f_C2func_getaddrinfo':
/tmp/go-build/cgo-gcc-prolog:58: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
# github.com/osrg/namazu/nmz
/tmp/go-link-045955592/000009.o: In function `mygetgrouplist':
/workdir/go/src/os/user/getgrouplist_unix.go:16: warning: Using 'getgrouplist' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrgid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:38: warning: Using 'getgrgid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:43: warning: Using 'getgrnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:33: warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwuid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:28: warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000004.o: In function `_cgo_26061493d47f_C2func_getaddrinfo':
/tmp/go-build/cgo-gcc-prolog:58: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
[INFO] Building nmz.static
# github.com/osrg/namazu/nmz
/tmp/go-link-045955592/000009.o: In function `mygetgrouplist':
/workdir/go/src/os/user/getgrouplist_unix.go:16: warning: Using 'getgrouplist' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrgid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:38: warning: Using 'getgrgid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetgrnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:43: warning: Using 'getgrnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:33: warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000008.o: In function `mygetpwuid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:28: warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-045955592/000004.o: In function `_cgo_26061493d47f_C2func_getaddrinfo':
/tmp/go-build/cgo-gcc-prolog:58: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
[INFO] Building example/template/mypolicy.go
[INFO] Building example/yarn/4301-reproduce/mypolicy.go
[INFO] Building Java Inspector (bin/nmz-inspector.jar)
[INFO] Building Java Analyzer (bin/nmz-analyzer.jar)
vagrant@ratis-build:~/namazu$ echo $?
0
```

I did find that RATIS-517 seems to have changed what JARs are loaded and so had to make a change for that after re-basing.

**Comment by joshelser** [comment]
Ok, let me try again. I thought this error'ed out for me. Maybe it was just me.

**Comment by joshelser** [comment]
Worked the second time around. Going to try once more for good measure.

The prepare namazu step failed the last time (I think it was something about the HDDs being slowed?). The error message didn't make sense to me. I'll make sure to grab it if it happens again.

**Comment by joshelser** [comment]
> Worked the second time around. Going to try once more for good measure.

Worked again. Merging.

Thanks for the changes, Clay.
